packages/styles/scss/patterns/sub-patterns/content-item/_content-item.scss

```diff
@@ -0,0 +1,40 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import '../../../globals/imports';
+
+@mixin content-item {
+  .#{$prefix}--content-item {
+    display: flex;
+    flex-direction: column;
+
+    &__heading {
+      margin-top: $carbon--spacing-07;
+      @include carbon--type-style('heading-02');
+    }
```

Comment:
A couple general comments on styling. We need to make sure we add the properly themed tokens like `$text-01` to the content. I also think we need to add the `95%` width here, and max-width of `640px`. As of right now that's pretty consistent across the board it looks like. This all goes for copy as well.

<img src="https://user-images.githubusercontent.com/3793636/73473243-0a746180-4352-11ea-8e40-40a09fabb6f6.png" width="400px" />

Reply:
done!